delphi-checks/src/test/java/au/com/integradev/delphi/checks/MixedNamesCheckTest.java

```diff
@@ -23,8 +23,30 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
+import org.sonar.plugins.communitydelphi.api.check.DelphiCheck;
 
 class MixedNamesCheckTest {
+  private static DelphiCheck createCheck() {
+    MixedNamesCheck check = new MixedNamesCheck();
+    check.excludedNames = "b_IgnoreBool";
```

Comment:
I'm not sure what this `b_` prefix is here - is it a Hungarian notation for boolean variables?

Since identifiers with this prefix is unconventional and would be caught by `ClassName`, `VariableName`, etc. it might make people reading this code wonder if there's a specific reason we're testing with this prefix.  Could we change this variable name to something else?